go.mod

```diff
@@ -1,114 +1,168 @@
 module github.com/hyperledger-labs/yui-relayer
 
-go 1.18
+go 1.19
```

Comment:
Looks like you use go1.20 in the test. Should we use the same version in the repo?

Reply:
Thank you. I've bumped up the versions.